web/components/ui/Content/Content.module.scss

```diff
@@ -13,20 +13,9 @@
   bottom: 0;
   width: 100%;
 
-	@include screen(tablet) {
-		top: 0;
+  @include screen(tablet) {
+    top: 0;
     position: relative;
-
-		&.online {
```

Comment:
wait do we still need to do this for other tabs?

Reply:
`&.online` exists for tablet, and I didn't want to touch anything tablet related because it seems to be working. Is there something I should look at?